azure/basic-arm-template/blueprint.yaml

```diff
@@ -0,0 +1,158 @@
+      - label: East US
+        value: eastus
+      - label: East US 2
+        value: eastus2
+      - label: West US
+        value: westus
+      - label: North Central US
+        value: northcentralus
+      - label: South Central US
+        value: southcentralus
+      - label: North Europe
+        value: northeurope
+      - label: West Europe
+        value: westeurope
+      - label: Japan West
+        value: japanwest
+      - label: Japan East
+        value: japaneast
+      - label: Brazil South
+        value: brazilsouth
+      - label: Australia East
+        value: australiaeast
+      - label: Australia Southeast
+        value: australiasoutheast
+      - label: South India
+        value: southindia
+      - label: Central India
+        value: centralindia
+      - label: West India
+        value: westindia
+      - label: Canada Central
+        value: canadacentral
+      - label: Canada East
+        value: canadaeast
+      - label: UK South
+        value: uksouth
+      - label: UK West
+        value: ukwest
+      - label: West Central US
+        value: westcentralus
+      - label: West US 2
+        value: westus2
+      - label: Korea Central
+        value: koreacentral
+      - label: Korea South
+        value: koreasouth
+      - label: France Central
+        value: francecentral
+      - label: France South
+        value: francesouth
+      - label: Australia Central
+        value: australiacentral
+      - label: Australia Central 2
+        value: australiacentral2
+      - label: South Africa North
+        value: southafricanorth
+      - label: South Africa West
+        value: southafricawest
+
+  - name: AppName
+    type: Input
+    prompt: What do you want to name the application?
+
+  - name: TemplateFile
+    type: Input
+    prompt: Path to the ARM template file
+    default: ../arm/template.json
+
+  - name: AdminUsername
+    type: Input
+    prompt: Admin username for logging in
+    default: azure
+
+  - name: AdminPassword
+    type: SecretInput
+    prompt: Admin password for logging in
+    description: "The supplied password must be between 6-72 characters long and must satisfy at least 3 of password complexity requirements from the following: (1) Contains an uppercase character, (2) Contains a lowercase character, (3) Contains a numeric digit, (4) Contains a special character, (5) Control characters are not allowed"
+    validate: !expr "strlen(AdminPassword) >= 6 && strlen(AdminPassword) <= 72"
+
+  - name: GenerateDockerComposeSetup
+    type: Confirm
+    prompt: "Do you want to generate a docker-compose setup with the required tools to run this blueprint?"
+    default: false
+
+  includeAfter:
+  - blueprint: xl-devops-platform
+    includeIf: GenerateDockerComposeSetup
+    parameterOverrides:
+    - name: UseXLDeploy
+      value: true
+    - name: XLDeployPort
+      value: 4516
+    - name: UseXLRelease
+      value: false
+    - name: XLVersion
+      value: "9.0.0"
```

Comment:
This needs to be changed to `9.0`

Reply:
Or `9.1` if arm plugin needs to be updated

Reply:
What's the next version that's being released? 9.5? I'll find out

Reply:
But I chatted with Deepu and there's definitely a consensus to have version numbers always be x.y.z, even in there are two zeroes

Reply:
I've updated it to 9.5.0 (the next release)